### DISCUSSION TITLE
Size of uploaded file exceeds 300MB

### DISCUSSION THREAD

**Discussion by AbdallahR77** [vercel/community#2994] [Help]
### Summary

Size of the uploaded file exceeds 300MB

### Steps to Reproduce

I am deploying a Flask application and I am using the GitHub repository but when I deploy  I face this problem
" Size of the uploaded file exceeds 300MB " but in my repo the biggest file is less than 50MB and these are all logs

-- | --
14:19:36.419 | Cloning 
14:19:36.660 | Previous build cache not available
14:19:36.864 | Cloning completed: 444.493ms
14:19:37.004 | Running "vercel build"
14:19:37.535 | Vercel CLI 30.2.3
14:19:37.662 | WARN! Due to `builds` existing in your configuration file, the Build and Development Settings defined in your Project Settings will not apply. Learn More: https://vercel.link/unused-build-settings
14:19:37.681 | Installing required dependencies...
14:22:06.380 | Build Completed in /vercel/output [2m]
14:22:08.465 | Deploying outputs...
14:52:40.384 | Error: Size of uploaded file exceeds 300MB
14:52:43.019 | : Size of uploaded file exceeds 300MB

**Comment by ghost** [comment]
I also have the same issue, any luck?

**Comment by salvere002** [comment]
same issue, my repo is less than 50MB

**Reply by GH0STH4CKER** [reply]
bcz modules

**Comment by ghost** [comment]
Could it be because I am installing tensorflow-cpu?

**Comment by ghost** [comment]
Could it be because I am installing tensorflow-cpu? Even then, it is very lightweight and none of the installation files are over 300MB

**Comment by shanbhxg** [comment]
having the same issue

**Comment by douglasichen** [comment]
Same issue. Has anyone found a solution yet?

**Comment by dauji** [comment]
same here, my entire git repo is less than 250mb, and I'm still getting this error

**Comment by Nemidas7734** [comment]
same issue

**Comment by douglasichen** [comment]
I had this issue and fixed it by switching to tensorflow.js.

**Comment by MrPotato-00** [comment]
Any update on this issue? How to solve this?

**Comment by aychoi** [comment]
Any news?
